client/my-sites/checkout/controller.jsx

```diff
@@ -121,45 +121,33 @@ export function gsuiteNudge( context, next ) {
 }
 
 export function conciergeSessionNudge( context, next ) {
```

Comment:
`conciergeSessionNudge` and `conciergeQuickstartSession` are almost identical.
Can we combine them into one?
"concierge session" was meant to be the internal name for these sessions, "quickstart" was really just a different way to talk about the concierge session in the copy. But the "quickstart" name has now spread pretty far.

I know you didn't introduce the duplication, and maybe it's out of scope for this. So feel free to ignore.

Reply:
I will take this as a follow up in a subsequent PR.